src/ILCompiler.Compiler/src/Compiler/CompilerTypeSystemContext.cs

```diff
@@ -231,7 +234,7 @@ private unsafe static PEReader OpenPEFile(string filePath, out MemoryMappedViewA
                 if (accessor != null)
                     accessor.Dispose();
                 if (mappedFile != null)
-                    mappedFile.Dispose();
+                    mappedFile.Dispose(); // Cleans up the fileStream.
```

Comment:
Jan, mappedFile closes the underlying fileStream. Not useful? I had to look up the code.